samples/cellular/nrf_cloud_multi_service/src/fota_support_coap.c

```diff
@@ -81,8 +81,8 @@ int coap_fota_thread_fn(void)
 		 */
 		err = nrf_cloud_fota_poll_process(&ctx);
 		if (err == -EAGAIN) {
-			LOG_DBG("Retrying in %d minute(s)",
-				CONFIG_COAP_FOTA_JOB_CHECK_RATE_MINUTES);
+			LOG_INF("Checking for FOTA job in %d seconds",
```

Comment:
why the change to display in seconds?

Reply:
To match the units of measurement used for the shadow logging interval